.github/workflows/ci.yml

```diff
@@ -16,6 +16,19 @@ jobs:
       # When running the container built on the CI
       # run: docker run -v $(pwd):/tmp/parsec -w /tmp/parsec -t parsec-service-test-all /tmp/parsec/ci.sh all
 
+  build-all-providers:
+    name: Runs all-providers cargo check (current Rust stable & old compiler)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      # Use the following step when updating the `parsec-service-test-all` image
+      # - name: Build the container
+      #   run: pushd e2e_tests/docker_image && docker build -t parsec-service-test-all -f parsec-service-test-all.Dockerfile . && popd
+      - name: Run the container to execute the test script
+        run: docker run -v $(pwd):/tmp/parsec -w /tmp/parsec ghcr.io/parallaxsecond/parsec-service-test-all /tmp/parsec/ci.sh cargo-check
+      # When running the container built on the CI
+      # run: docker run -v $(pwd):/tmp/parsec -w /tmp/parsec -t parsec-service-test-all /tmp/parsec/ci.sh build-all
```

Comment:
The argument of `ci.sh` here is `build-all` and not `cargo-check`, maybe because you changed your mind? I personally prefer `build-all` but will leave it to you 😄

Reply:
😂 we actually had a discussion about that, changed because I wasn't keen on `build-all`...

Reply:
I see 😛! Can stick to `cargo-check`